codenvy-api-project/src/main/java/com/codenvy/api/project/server/ProjectService.java

```diff
@@ -258,10 +258,17 @@ public ProjectDescriptor createProject(@ApiParam(value = "ID of workspace to cre
             throws ConflictException, ForbiddenException, ServerException {
 
         final GeneratorDescription generatorDescription = newProject.getGeneratorDescription();
+        Map<String, String> options;
+        if (generatorDescription == null) {
+            options = Collections.emptyMap();
+        } else {
+            options = generatorDescription.getOptions();
+        }
+
```

Comment:
com.google.common.base.MoreObjects.firstNonNull(generatorDescription.getOptions(), Collections.emptyMap())
Maybe this approach can be more clear WDYT?